app/controllers/api/person_escort_records_controller.rb

```diff
@@ -8,6 +8,11 @@ class PersonEscortRecordsController < ApiController
       relationships: [profile: {}],
     ].freeze
 
+    UPDATE_PERMITTED_PER_PARAMS = [
+      :type,
+      attributes: [:status],
+    ].freeze
+
     def create
       person_escort_record = PersonEscortRecord.save_with_responses!(
```

Comment:
should the `PersonEscortRecords::ParamsValidator` also be called on create?

Reply:
the validator is only specific to the update action, and not to the create values which are validated differently in the model as they are all relationships instead of attributes.